frontend/test/__support__/e2e/helpers/e2e-ad-hoc-question-helpers.js

```diff
@@ -6,6 +6,29 @@ export function adhocQuestionHash(question) {
   return btoa(unescape(encodeURIComponent(JSON.stringify(question))));
 }
 
-export function visitQuestionAdhoc(question) {
+export function visitQuestionAdhoc(question, { callback } = {}) {
```

Comment:
The next step will be to add `mode` to this helper because we can open ad questions in the simple mode, but also in a notebook mode.

That step is needed to replace `openTable()` helper function. You've seen it as `openOrdersTable()`,`openPeopleTable()`... We're deprecation `/question/new` route and these functions rely on it.

We can easily use `visitQuestionAdhoc`, instead.